docs/choreolib/getting-started.md

```diff
@@ -0,0 +1,165 @@
+# Getting Started
+
+## Installing ChoreoLib
+
+To use ChoreoLib in your robot code, it must first be installed as a vendor library. For more information about installing vendor libraries, see [WPILib's documentation](https://docs.wpilib.org/en/stable/docs/software/vscode-overview/3rd-party-libraries.html#installing-libraries) about installing in online mode.
+
+Use the following JSON URL to install ChoreoLib:
+
+=== "Release"
+
+    ```
+    https://lib.choreo.autos/dep/ChoreoLib.json
+    ```
+
+=== "2025 Beta"
+
+    ```
+    https://lib.choreo.autos/dep/ChoreoLib2025Beta.json
+    ```
+
+??? abstract "(Advanced) Building Manually"
```

Comment:
Maybe put this under contributing docs?

Reply:
There is currently a contributing guide but no information about setup/building, do you think there should be a separate contributing page in the ChoreoLib section about building with a link back to the main contributing guide, or should it be appended to the main article?